tests/testthat/test-geoknife_utils.R

```diff
@@ -46,7 +46,7 @@ test_that("check status of an ID that doesn't exist", {
 })
 
 fabric <- webdata('prism', variables = 'wrong')
-job <- geoknife(c(-89,43), fabric, wait = TRUE)
+suppressWarnings(job <- geoknife(c(-89,43), fabric, wait = TRUE))
```

Comment:
I don't think I actually need supressWarnings() here. With or without it, the code prints "General Error: Unable to open dataset at dods://cida.usgs.gov/thredds/dodsC/prism_v2 with identifier wrong"